src/Displaz.jl

```diff
@@ -6,6 +6,8 @@ using Colors
 export plot3d, plot3d!, clearplot, viewplot
 export KeyEvent, CursorPosition, event_loop
 
+displaz_cmd() = get(ENV, "DISPLAZ_CMD", "displaz")
```

Comment:
All add the other change. As for representing it as a function, there are two reasons: (1) it can be convenient not to have to reload the package if you forgot to set the variable before package loading; (2) I think there could be issues with precompilation (if that ever happened for this package), where the value at compile time would be frozen. 

A similar [choice](https://github.com/JuliaLang/julia/blob/5fe9e5342b06ef0cec58e67b9f7e4e82a9f0a860/base/client.jl#L54) was made for setting REPL colors in base.

Besides, it's not like there is a huge amount of overhead.

Still, it's not a big deal. If you'd prefer it to be a constant, I could live with that.